ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -350,11 +350,13 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8
-- as: nutanix-ipi-boot-categories-project-f28
+- as: nutanix-ipi-boot-categories-project-rt-f28
   cluster: build01
   cron: 12 22 4 * *
   steps:
     cluster_profile: nutanix-qe
+    env:
+      FIPS_ENABLED: "true"
```

Comment:
sorry, this is fip or rt?

Reply:
`rt` it is a copy/paste typo, later will update it.

Reply:
or can you help update it together in https://github.com/openshift/release/pull/54732

Reply:
It's not part of 4.13 job, I know @dis016 have work on 4.14, maybe he can help update it

Reply:
No problem, I will update it later.